Documentation/gettingstarted/servicemesh/installation.rst

```diff
@@ -0,0 +1,42 @@
+Prerequisites
+#############
+
+* Cilium must be configured with ``kubeProxyReplacement`` as partial
+  or strict.
```

Comment:
`partial` by itself with no other parameters means no KPR. Is there an additional dependency here?

Reply:
good point, let me add the link to our Kube proxy replacement docs.